React/Project structure.md

```diff
@@ -284,7 +284,60 @@ Example:
     └── WelcomeCard.elements.ts
 ```
 
-> Moving things to`.elements.tsx` should be the last step in the development process and it should only be used for organizational purposes, i.e. when the main component becomes cluttered and unreadable.
+In this example `WelcomeCardOverlay` is a pure Functional component because chakra factory doesn't allow custom `isOpen` prop.
+
+```tsx
+import { chakra, HTMLChakraProps, ThemingProps, useStyleConfig } from '@chakra-ui/react';
+
+export const WelcomeCardLayout = chakra(Grid, {
```

Comment:
In the [task](https://app.productive.io/1-infinum/tasks/2349078) you mentioned that using Chakra factory on Chakra components is an anti-pattern, yet this doesn't reflect that. Should this be changed?